source/_integrations/freedompro.markdown

```diff
@@ -0,0 +1,68 @@
+---
+title: Freedompro
+description: Instructions for how to integrate Freedompro accessories within Home Assistant.
+ha_category:
+  - Sensor
+  - Binary Sensor
+  - Climate
+  - Light
+  - Fan
+  - Lock
+  - Cover
+  - Switch
```

Comment:
The initial PR should be limited to a single platform.

Reply:
Our Products are bridges that control multiple type of accessories (light, switch, sensor,...). 
Should we remove all platforms except one for example light?

Reply:
Yeah starting with just a light, switch or a sensor is fine 👍 
See also the comment on the main PR, which has a reference to our main documentation.